DeDRM_plugin/adobekey.py

```diff
@@ -112,15 +111,13 @@ def unicode_argv():
             # Remove Python executable and commands if present
```

Comment:
It’s quite possibe that the entire `unicode_argv` function is not necessary anymore. A copy of this function exists in many modules, so this should be changed for all files in a different PR.